pages/book/cells.mdx

```diff
@@ -0,0 +1,371 @@
+# Cells, Builders and Slices
+
+import { Callout } from 'nextra/components'
+
+[Cells](#cells), [Builders](#builders) and [Slices](#slices) are low-level [primitives][p] of TON Blockchain. The virtual machine of TON Blockchain, [TVM][tvm], uses cells to represent all data structures in memory and persistent storage.
```

Comment:
>  uses cells to represent all data structures in memory

this is not correct: for instance, a tuple is a TVM primitive and it is also a data structure (although it is not directly storable)